lib/livebook_web/helpers.ex

```diff
@@ -165,7 +165,13 @@ defmodule LivebookWeb.Helpers do
     <div class="flex space-x-3 items-center justify-between">
       <span class="text-gray-700"><%= @label %></span>
       <label class={"switch-button #{if(@disabled, do: "switch-button--disabled")}"}>
-        <input class="switch-button__checkbox" type="checkbox" name={@name} checked={@checked} />
+        <input
+          data-element="input"
```

Comment:
This is specific to the input cell, so I think we should pass it from upstream. This can be achieved by passing all unknown attributes to the input element:

```elixir
assigns =
  ...
  |> assign(:attrs, assigns_to_attributes(assigns, [:label, :name, :checked, :disabled]))

# ...
~H"""
<input ... {@attrs} />
"""
```

With this change you should be able to do `<switch_checkbox data-element="input" />`

Reply:
Was wondering how to do this. Happy to see that it was implemented in 1.6.0-dev.